### DISCUSSION TITLE
[@types/chart.js] Cannot find module 'moment'

### DISCUSSION THREAD

**Discussion by pfroud** [DefinitelyTyped/DefinitelyTyped#59010] [Issues with a @types package] [answered]
Steps:

1. Create new folder.
2. `npm init`
3. `npm install --save @types/chart.js`
4. Create file tsconfig.json:
```
{
    "compilerOptions": {
        "target": "ES2021",
        "strict": true
    }
}
```
5. Create file main.ts:
```
const myChart = new Chart(document.createElement("canvas"), {});
```

When I run tsc, I get this error:
```
node_modules/@types/chart.js/index.d.ts:34:24 - error TS2792: Cannot find module 'moment'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?

34 import { Moment } from 'moment';
```

I zipped and attached the entire project folder. [chartjs-test.zip](https://github.com/DefinitelyTyped/DefinitelyTyped/files/8145999/chartjs-test.zip)

Environment:
* npm version 8.1.4
* tsc version 4.3.4

Refer to:
* https://github.com/DefinitelyTyped/DefinitelyTyped/issues/44423
* https://github.com/valor-software/ng2-charts/issues/1220

[a comment by an automated account is not shown]

**Comment by ricmello** [comment, marked as answer]
Setting `"moduleResolution": "node"` in tsconfig.json (which is recommended for most projects) should work.

**Comment by pfroud** [comment]
It works, thank you! I should've tried the advice in the error message, how embarrassing :)

```
Did you mean to set the 'moduleResolution' option to 'node'?
```

Links to documentation about the module resolution option:

https://www.typescriptlang.org/tsconfig#moduleResolution

https://www.typescriptlang.org/docs/handbook/module-resolution.html
